protocols/5842cf/Cherrypicking_and_Normalization.ot2.py

```diff
@@ -0,0 +1,109 @@
+from opentrons import labware, instruments, robot
+from otcustomizers import FileInput, StringSelection
+
+metadata = {
+    'protocolName': 'Cherrypicking Transfer and Sample Normalization from CSV',
+    'author': 'Chaz <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# load labware
+plate1X = labware.load(
+    'biorad_96_wellplate_200ul_pcr', '1', '1X DNA Plate')
+plate10X = labware.load(
+    'biorad_96_wellplate_200ul_pcr', '3', '1/10X DNA Plate')
+dest_plate = labware.load(
+    'biorad_96_wellplate_200ul_pcr', '2', 'Destination Plate')
+tips10 = [
+    labware.load('opentrons_96_tiprack_10ul', slot) for slot in ['4', '7']
+]
+tips50 = [
+    labware.load('opentrons_96_tiprack_300ul', slot) for slot in ['6', '9']
+]
+TrisHCl_container = labware.load(
+    'opentrons_10_tuberack_falcon_4x50ml_6x15ml_conical', '5',
+    '4-in-1 Tube Rack')
+
+TrisHCl = TrisHCl_container.wells('A3')
+
+example_csv = """
+A1,1/10X,0
+A2,1X,5.2
+A3,1/10X,50
+
+"""
+
+
+def run_custom_protocol(
+        picking_csv: FileInput = example_csv,
+        p10_mount: StringSelection('left', 'right') = 'left',
+        p50_mount: StringSelection('left', 'right') = 'right',
+        ):
+
+    # check
+    if p10_mount == p50_mount:
+        raise Exception('Input different mounts for P10 and P50 pipettes')
+
+    # create pipettes
+    pip10 = instruments.P10_Single(mount=p10_mount, tip_racks=tips10)
+    pip50 = instruments.P50_Single(mount=p50_mount, tip_racks=tips50)
+
+    tip10_max = len(tips10)*96
+    tip50_max = len(tips50)*96
+    tip10_count = 0
+    tip50_count = 0
+
+    def pick_up(pip):
+        nonlocal tip10_count
+        nonlocal tip50_count
+
+        if pip == 'pip50':
```

Comment:
You can just compare `pip` to the pipette object `pip50` rather than the string `'pip50'`. Meaning that you can just pass the pipette object as the argument. You can also refactor this function a bit to simplify it, see the comment below